source/class/qx/lang/Type.js

```diff
@@ -52,15 +52,23 @@ qx.Bootstrap.define("qx.lang.Type", {
     isArray: qx.Bootstrap.isArray,
 
     /**
-     * Whether the value is an object. Note that built-in types like Window are
-     * not reported to be objects.
+     * Whether the value is an object i.e. Object.prototype is its prototype or Object.prototype is in its prototype chain.
+     * Note that built-in types like Window are not reported to be objects.
      *
      * @signature function(value)
-     * @param value {var} Value to check.
+     * @param {*} value value to check.
      * @return {Boolean} Whether the value is an object.
      */
     isObject: qx.Bootstrap.isObject,
 
+    /**
```

Comment:
Same as above, the net effect is more useful in documentation than repeating the code, eg "Detects whether an object is just a plain POJO, ie `{}`; note that this excludes objects created via classes - @see `qx.lang.Type.isObject`